docker/templates/new-django-app/{{cookiecutter.app_name}}/{{cookiecutter.module_name}}/static/js/base.js

```diff
@@ -1,3 +1,3 @@
-import { helloWorld } from "./helloworld";
+import { helloWorld } from "./helloworld"
 
-helloWorld();
```

Comment:
I've been doing this lately too ! Seems a little bit cleaner with ES6 syntax/features

Reply:
Agree! In this case, our linting rules will also complain about the semicolons, so I had to nix them for the tests to pass. 😅 Feels great to take a principled stance on syntax that until now has been pretty idiosyncratic.